extensions/amp-list/0.1/amp-list.js

```diff
@@ -410,6 +412,27 @@ export class AmpList extends AMP.BaseElement {
     return startsWith(src, AMP_STATE_URI_SCHEME);
   }
 
+  /**
+   * Returns true if element's src points to an amp-script function.
+   *
+   * @param {string} src
+   * @return {boolean}
+   * @private
+   */
+  isAmpScriptSrc_(src) {
+    if (getMode().development) {
+      devAssert(
```

Comment:
Experiment assertions usually look like:
```js
 userAssert(/* experiment condition */, `Experiment ${EXPERIMENT} is not turned on.`);
```
I don't think we need to check the mode.

Reply:
Hmm. I'd be able to write it like that if I moved it to the `getAmpScriptSrc` function instead of the detector. Updating!

Reply:
Meh. Couldn't get it as clean as a single line assertion since it needed to be wrapped in a Promise, but its def a bit better in its new home. Thanks for the pointer!